### PR TITLE
EWS: Add missing awaits

### PR DESCRIPTION
**Pull request by jermy-c**

### CODE REVIEW

**Review by benbucksch — APPROVED** [review]

File: app/logic/Mail/EWS/EWSAccount.ts [hunk 1]
```diff
@@ -360,7 +360,7 @@ export class EWSAccount extends MailAccount {
         m$ConnectionTimeout: 29, // minutes
       },
     };
-    this.callStream(streamRequest, async message => {
+    await this.callStream(streamRequest, async message => {
```
Comment (NeilRashbrook) [inline app/logic/Mail/EWS/EWSAccount.ts:363]: This was deliberately not an `await`, because it takes 30 minutes to complete.

Comment (benbucksch) [reply app/logic/Mail/EWS/EWSAccount.ts:363]: Ah, thanks for the note. We'll change that to `this.callStream(...).catch(this.errorCallback);`

@NeilRashbrook  Having neither of those is *always* a bug. It leads to uncaught exceptions, and even to crashes. So, every `async` function call (and every promise run) *must* be either with `await` or with `.catch()`.

Comment (NeilRashbrook) [reply app/logic/Mail/EWS/EWSAccount.ts:363]: `callStream` already has its own `try/catch`.

Comment (benbucksch) [reply app/logic/Mail/EWS/EWSAccount.ts:363]: Fixed in git commit 9cd12445

Comment (benbucksch) [reply app/logic/Mail/EWS/EWSAccount.ts:363]: @NeilRashbrook Thanks for alerting about this. Indeed, `await` for 29 minutes would have been bad here.

**Pull request merged**